website/source/api/acl/roles.html.md

```diff
@@ -0,0 +1,462 @@
+---
+layout: api
+page_title: ACL Roles - HTTP API
+sidebar_current: api-acl-roles
+description: |-
+  The /acl/role endpoints manage Consul's ACL Roles.
+---
+
+-> **1.5.0+:**  The APIs are available in Consul versions 1.5.0 and later. The documentation for the legacy ACL API is [here](/api/acl/legacy.html)
+
+# ACL Role HTTP API
+
+The `/acl/role` endpoints [create](#create-a-role), [read](#read-a-role),
+[update](#update-a-role), [list](#list-roles) and [delete](#delete-a-role)  ACL roles in Consul.
+For more information about ACLs, please see the [ACL Guide](/docs/guides/acl.html).
+
+## Create a Role
+
+This endpoint creates a new ACL role.
+
+| Method | Path                         | Produces                   |
+| ------ | ---------------------------- | -------------------------- |
+| `PUT`  | `/acl/role`                  | `application/json`         |
+
+The table below shows this endpoint's support for
+[blocking queries](/api/index.html#blocking-queries),
+[consistency modes](/api/index.html#consistency-modes),
+[agent caching](/api/index.html#agent-caching), and
+[required ACLs](/api/index.html#acls).
+
+| Blocking Queries | Consistency Modes | Agent Caching | ACL Required |
+| ---------------- | ----------------- | ------------- | ------------ |
+| `NO`             | `none`            | `none`        | `acl:write`  |
+
+### Parameters
+
+- `Name` `(string: <required>)` - Specifies a name for the ACL role. The name
+   can only contain alphanumeric characters as well as `-` and `_` and must be
```

Comment:
```suggestion
   can only contain alphanumeric characters, dashes `-` , underscores `_`,  and must be
```
If someone is using a screen reader it probably won't recognize just `_` or `-`

Reply:
That's actually text I cloned from the policies page (so it might be worth updating the same thing over there unrelated to my PR). I left a TODO because I need to update the validation text as it is more restrictive than policy names.

Reply:
I think in this case its ok to be inconsistent for now, until we make bigger updates. 

We should maybe use an accessibility tool to catch all these things on the site and then make one big update.....